go-controller/pkg/node/gateway_init.go

```diff
@@ -530,9 +530,6 @@ func CleanupClusterNode(name string) error {
 		klog.Errorf("Failed to delete ovn-bridge-mappings, stdout: %q, stderr: %q, error: %v", stdout, stderr, err)
 	}
 
-	// Delete iptable rules for management port
-	DelMgtPortIptRules()
```

Comment:
I can't find where the upgrade is handled: we should delete these rules before switching to nftables?

Reply:
hm... probably I guess